care_scribe/tasks/scribe.py

```diff
@@ -501,33 +580,40 @@ def generate_response(retry=0):
             form.chat_output_tokens = ai_response.usage.completion_tokens
 
         logger.info(f"AI response: {ai_response_json}")
+        processing["completion_time"] = perf_counter() - completion_start_time
+
+        # convert the keys back to the original field IDs
+        converted_response = {
+            k: ai_response_json.get(f"q{i}")
+            for i, k in enumerate(processed_fields)
+            if ai_response_json.get(f"q{i}") is not None
+        }
+        form.ai_response = converted_response
+        processing["ai_response"] = converted_response
+        form.meta["processings"] = [*form.meta.get("processings", []), processing]
+        form.status = Scribe.Status.COMPLETED
+        form.save()
+
+        logger.info(f"Scribe[{form.external_id}] status: completed AI form processing")
+
+        # Update the user and facility quotas
+        if not is_benchmark:
+            user_quota.calculate_used()
+            facility_quota.calculate_used()
 
     except Exception as e:
-        # Log the error or handle it as needed
-        logger.error(f"AI form fill processing failed at line {e.__traceback__.tb_lineno}: {e}")
+        logger.error(f"Scribe[{form.external_id}] status: error occurred while processing form: {e}")
+
+        if getattr(settings, "SENTRY_DSN", None):
+            import sentry_sdk
+
+            sentry_sdk.capture_exception(e)
```

Comment:
Question: Last I checked, logger.error was feeding into Sentry, and I could see logs on sentry... so why are we explicitly calling it again? Unless something on core was changed.